### ISSUE TITLE
terms page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#290]
```
it shows an error:

Traceback (most recent call last):
  File "/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/__init__.py", line 511, in __call__
    handler.get(*groups)
  File "/base/data/home/apps/html5rocks/3.345500695094989155/main.py", line 176, in get
    self.render(template_path=path)
  File "/base/data/home/apps/html5rocks/3.345500695094989155/main.py", line 117, in render
    'toc' : self.get_toc(template_path),
  File "/base/data/home/apps/html5rocks/3.345500695094989155/main.py", line 45, in get_toc
    template_text = webapp.template.render(path, {});
  File "/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py", line 81, in render
    return t.render(Context(template_dict))
  File "/base/python_runtime/python_lib/versions/1/google/appengine/ext/webapp/template.py", line 121, in wrap_render
    return orig_render(context)
  File "/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py", line 168, in render
    return self.nodelist.render(context)
  File "/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py", line 705, in render
    bits.append(self.render_node(node, context))
  File "/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/__init__.py", line 718, in render_node
    return(node.render(context))
  File "/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader_tags.py", line 63, in render
    compiled_parent = self.get_parent(context)
  File "/base/python_runtime/python_lib/versions/third_party/django-0.96/django/template/loader_tags.py", line 58, in get_parent
    raise TemplateSyntaxError, "Template %r cannot be extended, because it doesn't exist" % parent
TemplateSyntaxError: Template 'base.html' cannot be extended, because it 
doesn't exist

```

Original issue reported on code.google.com by `antonino...@gmail.com` on 16 Nov 2010 at 9:44

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision d24ce8895c.
```

Original comment by `ericbide...@html5rocks.com` on 16 Nov 2010 at 7:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 16 Nov 2010 at 7:20
- Added labels: **Milestone-4**
